packages/ui/src/accounts/components/AccountItem/components/lockItems/CouncilCandidateLockItem.tsx

```diff
@@ -25,26 +27,41 @@ export const CouncilCandidateLockItem = ({ lock, address, isRecoverable }: LockD
   const memberId = useMemo(() => getMemberIdByBoundAccountAddress(address), [address])
   const { candidateId } = useCandidateIdByMember(memberId || '-1')
   const { data } = useGetNewCandidateEventsQuery({ variables: { lockAccount: address } })
+  const { stage: electionStage } = useElectionStage()
+  const remainingPeriod = useCouncilRemainingPeriod()
 
   const eventData = data?.newCandidateEvents[0]
   const createdInEvent = eventData && asBlock(eventData)
 
   const electionId = eventData?.candidate.electionRoundId
-  const electionStart = eventData?.candidate.electionRound.referendumStageVoting?.createdAt
   const voteStageDuration = api?.consts.referendum?.voteStageDuration.toNumber()
   const revealStageDuration = api?.consts.referendum?.revealStageDuration.toNumber()
 
   const recoveryTime = useMemo(() => {
-    if (!electionStart || !voteStageDuration || !revealStageDuration) {
-      return
+    if (!voteStageDuration || !revealStageDuration || !remainingPeriod) {
+      return { unrecoverableLabel: 'Recoverable after failed election' }
```

Comment:
Shouldn't it be something like: "Recoverable after lost election or end of the council term" ?

Reply:
Not really, council candidate lock can only be withdrawn after the lost election, if one wins the election then the lock transforms to a councilor lock which is handled by another list item.

This other list item for Councilor has the following tooltip:
<img width="626" alt="image" src="https://user-images.githubusercontent.com/92513933/192706862-3c3fd0ff-e80c-4465-9e20-edec62a4d29d.png">